i18n/locales/chinese/translations.json

```diff
@@ -16,31 +16,23 @@
     "show-comments": "显示评论",
```

Comment:
This will force other than standard ( `UTF-*` ) code-page.

Reply:
I'm not sure what you mean. We use i18next for localization during the build process, and switch out strings based on the files in the `locales/lang` directories.

Everything is still UTF-8 encoded once the pages are built.